tools/docker/sanity_check.Dockerfile

```diff
@@ -106,6 +106,30 @@ RUN TF_ADDONS_NO_BUILD=1 pip install --no-deps -e .
 RUN python tools/docs/build_docs.py
 RUN touch /ok.txt
 
+# -------------------------------
+# docs tests
+FROM python:3.6 as test_editable_mode
+
+COPY build_deps/build-requirements-cpu.txt ./
+RUN pip install -r build-requirements-cpu.txt
+COPY requirements.txt ./
+RUN pip install -r requirements.txt
+
+RUN apt-get update && apt-get install -y sudo rsync
+COPY tools/ci_build/install/bazel.sh ./
+RUN bash bazel.sh
+COPY tools/docker/finish_bazel_install.sh ./
+RUN bash finish_bazel_install.sh
+
+
+COPY ./ /addons
+WORKDIR /addons
+RUN python configure.py --no-deps
+RUN bash tools/install_so_files.sh
+RUN TF_ADDONS_NO_BUILD=1 pip install --no-deps -e .
+RUN python -c "from tensorflow_addons.activations import gelu; print(gelu(0.2))"
```

Comment:
Same as the other PR could we switch to lisht

Reply:
Sure thing